CHANGELOG.rst

```diff
@@ -9,13 +9,22 @@ This document describes changes between each past release.
 
 **Protocol**
 
-* ``_since`` and ``_before`` now accepts an integer value between quotes ``"``,
+- ``_since`` and ``_before`` now accepts an integer value between quotes ``"``,
   as it would be returned in the ``ETag`` response header.
+- A batch request now fails if one of the subrequests fails (#510)
+  (*see new feature about transactions*)
+
+**New features**
+
+- A transaction now covers the whole request/response cycle (#510, Kinto/kinto#194).
+  If an error occurs during the request processing, every operation performed
+  is rolled back. **Note:** This is only enabled with *PostgreSQL* backends.
```

Comment:
what happens when I have a postgres backend for the storage and redis for the cache ? can you confirm cache changes are also rollbacked ?